wrappers/services.go

```diff
@@ -456,21 +461,22 @@ func EnableSnapServices(s *snap.Info, inter interacter) (err error) {
 	return nil
 }
 
-// StopFlags carries extra flags for StopServices.
-// It reflects backend.StopFlags.
-type StopFlags struct {
+// StopServicesFlags carries extra flags for StopServices.
+type StopServicesFlags struct {
 	Disable bool
 }
 
 // StopServices stops and optionally disables service units for the applications
 // from the snap which are services.
-func StopServices(apps []*snap.AppInfo, flags *StopFlags, reason snap.ServiceStopReason, inter interacter, tm timings.Measurer) error {
+func StopServices(apps []*snap.AppInfo, flags *StopServicesFlags, reason snap.ServiceStopReason, inter interacter, tm timings.Measurer) error {
 	sysd := systemd.New(dirs.GlobalRootDir, systemd.SystemMode, inter)
 	if flags == nil {
-		flags = &StopFlags{}
+		flags = &StopServicesFlags{}
 	}
 
-	logger.Debugf("StopServices called for %q, reason: %v", apps, reason)
+	if reason != snap.StopReasonOther {
```

Comment:
we probably want to still log something but just without the reason: bit

Reply:
Sure, will do. Thanks